src/main/java/roomescape/controller/ReservationTimeController.java

```diff
@@ -0,0 +1,43 @@
+package roomescape.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.dto.ReservationTimeRequestDto;
+import roomescape.dto.ReservationTimeResponseDto;
+import roomescape.service.ReservationTimeService;
+
+@RestController
```

Comment:
아래와 같이 작성하면 혹시 모를 실수를 방지할 수 있겠네요!
ReservationController 도 함께 챙겨주세요 🙂 

```suggestion
@RestController
@RequestMapping("/times")
```